lib/perl/Genome/Model/Tools/Varscan/SomaticFilterWorkflow.pm

```diff
@@ -116,52 +115,65 @@ sub check_result {
     my $self = shift;
     my $result = shift;
     unless($result){
-        foreach my $error (@Workflow::Simple::ERROR){
-            print STDERR $error->error ."\n";
-        }
         die $self->error_message("Workflow did not return correctly.");
     }
 }
 
-sub validate_workflow {
+sub execute_workflow_for_operation {
     my $self = shift;
-    my $w = shift;
-    my @errors = $w->validate;
-    if (@errors) {
-        $self->error_message(@errors);
-        die "Errors validating workflow\n";
+    my $op = shift;
+    my $op_shortname = shift;
+    my $inputs = shift;
+
+    my $workflow = Genome::WorkflowBuilder::DAG->create(
+        name => $op_shortname . ' workflow',
+    );
+    $workflow->add_operation($op);
+
+    for my $input (keys %$inputs) {
+        $workflow->connect_input(
+            input_property => $input,
+            destination => $op,
+            destination_property => $input,
+        );
     }
+
+    $workflow->connect_output(
+        source => $op,
+        source_property => 'result',
+        output_property => 'result',
+    );
+
+    $workflow->recursively_set_log_dir($self->outdir);
+
+    $self->status_message("Launching %s workflow now.", $op_shortname);
+    my $result = $workflow->execute(inputs => $inputs);
+    return $result;
 }
 
 sub set_lsf_queue {
     my $self = shift;
     my $w = shift;
     my $lsf_queue = Genome::Config::get('lsf_queue_build_worker_alt');
-    $w->operation_type->lsf_queue($lsf_queue);
+    $w->lsf_queue($lsf_queue);
 }
 
 sub run_format_workflow {
     my $self = shift;
     my $input = shift;
 
-    my $w = Workflow::Operation->create(
+    my $w = Genome::WorkflowBuilder::Command->create(
         name => "Format Varscan SNVs",
-        operation_type => Workflow::OperationType::Command->get(
-            'Genome::Model::Tools::Capture::FormatSnvs')
+        command => 'Genome::Model::Tools::Capture::FormatSnvs',
     );
     $w->parallel_by('variants_file');
     $self->set_lsf_queue($w);
-    $w->log_dir($self->outdir);
-    $self->validate_workflow($w);
 
-    # Launch workflow
-    $self->status_message("Launching format workflow now.");
-    my $result = Workflow::Simple::run_workflow_lsf(
-        $w,
+    my $result = $self->execute_workflow_for_operation($w, 'format', {
```

Comment:
Perhaps outside the scope of this PR, but I think using named arguments here would be clearer.